### ISSUE TITLE
Responsive Editor bug when Search for a Widget

### ISSUE THREAD

**Issue by maurocarreraok** [elementor/elementor#14026]
## Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest development version of Elementor.

## Description

When you search for a Widget on the new version of the Editor the items shows 1 per row instead of alining like the responsive normaly does.

Normal situation:
![image](https://user-images.githubusercontent.com/68023943/109029169-53a3fb80-76a1-11eb-9a22-4d8a5cd567ec.png)

When search:
![image](https://user-images.githubusercontent.com/68023943/109029199-5bfc3680-76a1-11eb-8d02-468186ae0838.png)

## Environment

<details>
<summary>System Info</summary>
```
					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: Source distribution v5.7.32-35
	PHP Version: 7.3.27
	PHP Max Input Vars: 3000
	PHP Max Post Size: 256M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.6.2
	Site URL: https://elementor.maurocarrera.com
	Home URL: https://elementor.maurocarrera.com
	WP Multisite: No
	Max Upload Size: 256 MB
	Memory limit: 768M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.3.1
	Author: Elementor Team
	Child Theme: No

== User ==
	Role: administrator
	WP Profile lang: en_US
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/88.0.4324.182 Safari/537.36

== Active Plugins ==
	Classic Editor
		Version: 1.6
		Author: WordPress Contributors

	Crocoblock Wizard
		Version: 1.1.0
		Author: Crocoblock

	Elementor
		Version: 3.2.0-dev4
		Author: Elementor.com

	Elementor Beta (Developer Edition)
		Version: 1.1.1
		Author: Elementor.com

	Elementor Pro
		Version: 3.2.0-dev1
		Author: Elementor.com

	Jet Appointments Booking
		Version: 1.3.2
		Author: Crocoblock

	JetBlocks For Elementor
		Version: 1.2.7
		Author: Crocoblock

	JetBlog For Elementor
		Version: 2.2.12
		Author: Crocoblock

	JetBooking
		Version: 2.2.1
		Author: Crocoblock

	JetCompareWishlist For Elementor
		Version: 1.2.4
		Author: Crocoblock

	JetElements For Elementor
		Version: 2.5.4
		Author: Crocoblock

	JetEngine
		Version: 2.7.2
		Author: Crocoblock

	JetGridBuilder
		Version: 1.1.0
		Author: Crocoblock

	JetMenu
		Version: 2.0.8
		Author: Crocoblock

	JetPopup
		Version: 1.5.1
		Author: Crocoblock

	JetProductGallery For Elementor
		Version: 1.2.0
		Author: Crocoblock

	JetReviews For Elementor
		Version: 2.1.0
		Author: Crocoblock

	JetSearch For Elementor
		Version: 2.1.9
		Author: Crocoblock

	JetSmartFilters
		Version: 2.1.1
		Author: Crocoblock

	JetTabs For Elementor
		Version: 2.1.10
		Author: Crocoblock

	JetThemeCore
		Version: 1.2.1
		Author: Crocoblock

	JetTricks
		Version: 1.3.4
		Author: Crocoblock

	JetWooBuilder For Elementor
		Version: 1.7.6
		Author: Crocoblock

	OoohBoi Steroids for Elementor
		Version: 1.6.2
		Author: OoohBoi

	Ultimate Addons for Elementor
		Version: 1.29.1
		Author: Brainstorm Force

	WooCommerce
		Version: 5.0.0
		Author: Automattic

	WooCommerce Product Generator
		Version: 1.1.1
		Author: itthinx


== Elementor - Compatibility Tag ==
	
	Elementor Pro: incompatible
	JetBlocks For Elementor: header_not_exists
	JetBlog For Elementor: header_not_exists
	JetCompareWishlist For Elementor: header_not_exists
	JetElements For Elementor: header_not_exists
	JetProductGallery For Elementor: header_not_exists
	JetReviews For Elementor: header_not_exists
	JetSearch For Elementor: header_not_exists
	JetTabs For Elementor: header_not_exists
	JetWooBuilder For Elementor: header_not_exists
	OoohBoi Steroids for Elementor: header_not_exists
	Ultimate Addons for Elementor: incompatible

== Elementor Pro - Compatibility Tag ==
	
	Ultimate Addons for Elementor: incompatible

== Elements Usage ==
	
	jet-woo-builder-checkout : 0
		jet-checkout-billing : 1
		jet-checkout-coupon-form : 1
		jet-checkout-order-review : 1
		jet-checkout-payment : 1
		jet-checkout-shipping-form : 1


== Log ==
	
PHP: showing 5 of 5PHP: 2020-08-30 18:27:38 [notice X 2][/home/customer/www/elementor.maurocarrera.com/public_html/wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-11-26 07:36:36 [notice X 1][/home/customer/www/elementor.maurocarrera.com/public_html/wp-content/plugins/elementor/includes/template-library/classes/class-import-images.php::154] Undefined index: url [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-12-13 18:37:05 [notice X 14][/home/customer/www/elementor.maurocarrera.com/public_html/wp-content/plugins/elementor-pro/modules/posts/widgets/posts.php::31] Undefined index: posts_post_type [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2020-12-14 17:06:14 [notice X 11][/home/customer/www/elementor.maurocarrera.com/public_html/wp-content/plugins/elementor-pro/modules/query-control/classes/elementor-post-query.php::226] Undefined property: WP_Error::$taxonomy [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2021-01-03 18:50:22 [notice X 2][/home/customer/www/elementor.maurocarrera.com/public_html/wp-content/plugins/elementor/includes/template-library/sources/local.php::705] ob_end_clean(): failed to delete buffer. No buffer to delete [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]

Log: showing 20 of 312020-12-17 15:04:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.0.14',
  'to' => '3.1.0-dev2',
)]
2020-12-18 13:40:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.0-dev2',
  'to' => '3.1.0-dev3',
)]
2020-12-21 20:22:45 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.0-dev3',
  'to' => '3.1.0-dev4',
)]
2020-12-30 16:46:26 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.8',
  'to' => '3.0.9',
)]
2021-01-09 02:44:57 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.1.0-dev4',
  'to' => '3.2.0-dev1',
)]
2021-01-21 15:31:09 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.0.9',
  'to' => '3.1.0-dev1',
)]
2021-01-27 19:07:17 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-dev1',
  'to' => '3.2.0-dev2',
)]
2021-02-09 13:43:40 [info] elementor-pro::elementor_pro_updater Started 
2021-02-09 13:43:40 [info] Elementor Pro/Upgrades - _v_3_1_0_media_carousel Start  
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_media_carousel Finished 
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_reviews Start  
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_reviews Finished 
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_testimonial_carousel Start  
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_testimonial_carousel Finished 
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_slides Start  
2021-02-09 13:43:41 [info] Elementor Pro/Upgrades - _v_3_1_0_slides Finished 
2021-02-09 13:43:41 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.1.0-dev1',
  'to' => '3.2.0-dev1',
)]
2021-02-09 13:43:41 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.1.0-dev1',
  'to' => '3.2.0-dev1',
)]
2021-02-19 12:28:06 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-dev2',
  'to' => '3.2.0-dev3',
)]
2021-02-22 21:56:40 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.2.0-dev3',
  'to' => '3.2.0-dev4',
)]

JS: showing 5 of 5JS: 2020-12-17 15:04:51 [error X 17][https://elementor.maurocarrera.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils&ver=5.6:2:31670] Cannot read property \'landingPagesHasPages\' of undefined 
JS: 2020-12-17 15:04:51 [error X 10][https://elementor.maurocarrera.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils&ver=5.6:2:31670] elementorAdmin is not defined 
JS: 2020-12-17 15:05:12 [error X 1][https://elementor.maurocarrera.com/wp-content/plugins/elementor/assets/js/beta-tester.min.js?ver=3.1.0-dev2:2:23495] elementorBetaTester is not defined 
JS: 2020-12-17 15:13:13 [error X 2][https://elementor.maurocarrera.com/wp-includes/js/jquery/jquery.min.js?ver=3.5.1:2:31670] Cannot read property \'elementWrappers\' of undefined 
JS: 2020-12-18 14:27:34 [error X 3][https://elementor.maurocarrera.com/wp-admin/load-scripts.php?c=0&loadchunk_0=jquery-core,jquery-migrate,utils&ver=5.6:2:31670] Cannot read property \'option_enabled\' of undefined 


				
```
</details>

**Comment by fabianenzensperger** [comment]
Yes me too. Believe it shouldn't be like that! 😅

**Comment by DorShahar** [comment]
Hey all,

This issue has been resolved in Elementor v3.2.0

Feel free to update!
